yarn.lock

```diff
@@ -4946,42 +5118,43 @@ jest-watch-typeahead@^0.5.0:
     string-length "^3.1.0"
     strip-ansi "^6.0.0"
 
-jest-watcher@^25.2.4, jest-watcher@^25.3.0:
-  version "25.3.0"
-  resolved "https://registry.yarnpkg.com/jest-watcher/-/jest-watcher-25.3.0.tgz#fd03fd5ca52f02bd3161ab177466bf1bfdd34e5c"
-  integrity sha512-dtFkfidFCS9Ucv8azOg2hkiY3sgJEHeTLtGFHS+jfBEE7eRtrO6+2r1BokyDkaG2FOD7485r/SgpC1MFAENfeA==
+jest-watcher@^25.2.4, jest-watcher@^25.5.0:
+  version "25.5.0"
+  resolved "https://registry.yarnpkg.com/jest-watcher/-/jest-watcher-25.5.0.tgz#d6110d101df98badebe435003956fd4a465e8456"
+  integrity sha512-XrSfJnVASEl+5+bb51V0Q7WQx65dTSk7NL4yDdVjPnRNpM0hG+ncFmDYJo9O8jaSRcAitVbuVawyXCRoxGrT5Q==
   dependencies:
-    "@jest/test-result" "^25.3.0"
-    "@jest/types" "^25.3.0"
+    "@jest/test-result" "^25.5.0"
+    "@jest/types" "^25.5.0"
     ansi-escapes "^4.2.1"
     chalk "^3.0.0"
-    jest-util "^25.3.0"
+    jest-util "^25.5.0"
     string-length "^3.1.0"
 
-jest-worker@^24.6.0:
-  version "24.9.0"
-  resolved "https://registry.yarnpkg.com/jest-worker/-/jest-worker-24.9.0.tgz#5dbfdb5b2d322e98567898238a9697bcce67b3e5"
-  integrity sha512-51PE4haMSXcHohnSMdM42anbvZANYTqMrr52tVKPqqsPJMzoP6FYYDVqahX/HrAoKEKz3uUPzSvKs9A3qR4iVw==
+jest-worker@^25.5.0:
+  version "25.5.0"
+  resolved "https://registry.yarnpkg.com/jest-worker/-/jest-worker-25.5.0.tgz#2611d071b79cea0f43ee57a3d118593ac1547db1"
+  integrity sha512-/dsSmUkIy5EBGfv/IjjqmFxrNAUpBERfGs1oHROyD7yxjG/w+t0GOJDX8O1k32ySmd7+a5IhnJU2qQFcJ4n1vw==
   dependencies:
     merge-stream "^2.0.0"
-    supports-color "^6.1.0"
+    supports-color "^7.0.0"
 
-jest-worker@^25.2.6:
-  version "25.2.6"
-  resolved "https://registry.yarnpkg.com/jest-worker/-/jest-worker-25.2.6.tgz#d1292625326794ce187c38f51109faced3846c58"
-  integrity sha512-FJn9XDUSxcOR4cwDzRfL1z56rUofNTFs539FGASpd50RHdb6EVkhxQqktodW2mI49l+W3H+tFJDotCHUQF6dmA==
+jest-worker@^26.2.1:
+  version "26.3.0"
+  resolved "https://registry.yarnpkg.com/jest-worker/-/jest-worker-26.3.0.tgz#7c8a97e4f4364b4f05ed8bca8ca0c24de091871f"
+  integrity sha512-Vmpn2F6IASefL+DVBhPzI2J9/GJUsqzomdeN+P+dK8/jKxbh8R3BtFnx3FIta7wYlPU62cpJMJQo4kuOowcMnw==
   dependencies:
+    "@types/node" "*"
     merge-stream "^2.0.0"
     supports-color "^7.0.0"
 
 jest@^25.3.0:
-  version "25.3.0"
```

Comment:
I'm not sure why this was changed as it seems unrelated and I couldn't find a dep that was upgraded that has `jest` itself as direct dependency nor one that was updated.

This and a few other changes, like to `@babel/core` make me suspect you ran a `yarn upgrade` which I do not recommend as it just upgrades everything -- including minors and patches unnecessarily -- instead of the exact dependencies that were changed or need upgrading.

If so, I would throw out these changes to `yarn.lock` and just run `yarn` + `yarn deduplicate` against the new `package.json`, which should condense to only the changes that are necessary.

Reply:
If I understand correctly, I should revert `yarn.lock` to how it looks on the current master, then run the `yarn` commands? I don’t think I ran `yarn upgrade` but I am more familiar with `npm` than `yarn` so I could have made some mistake.

Reply:
> I should revert `yarn.lock` to how it looks on the current master, then run the `yarn` commands?

Yep, exactly.

> I don’t think I ran `yarn upgrade` but I am more familiar with `npm` than `yarn` so I could have made some mistake.

Yea same. `npm upgrade rollup@"^2.28.2"` I believe would only update Rollup and anything affected, whereas `yarn upgrade rollup@"^2.28.2"` basically ignores the argument, at least in my experience. `yarn add rollup@"^2.28.2"` I _think_ only upgrades it, but there's some quirks I've found with that as well. `yarn-deduplicate` itself exists because of a bug/bugs with Yarn that it doesn't dedupe properly (despite `yarn dedupe` saying otherwise). 

Upgrading `package.json` by hand and then running `yarn` + `yarn deduplicate` has been the safest and most predictable method I've found of updating the lockfile 😅

Reply:
`rollup-plugin-terser` depends on `jest-worker`

Reply:
Yes, but `jest-worker` doesn't depend on `jest` itself (which can be seen above this line), so that shouldn't have changed.

It looks like you followed the directions in your latest update to the PR and now `yarn.lock` looks correctly changed. It's about 10% the changes of the previous version.